assets/js/previews/cart-items.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import productPicture from './product-image';
+
+// Sample data for cart block line items.
+// This closely resembles the data returned from the Store API cart/items endpoint.
+// https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/master/src/RestApi/StoreApi#cart-items-api
+export const previewCartItems = [
+	{
```

Comment:
Curious - I was using an example response from the API as reference (on current branch), not the schema itself. It had `srcset` etc key in `images` array, and all the `currency_*` keys in `totals` object. I'm guessing the API is returning more useful stuff and the schema needs updating (@mikejolley ?). 

In any case this is hard-coded temporary data, so hopefully not blocking merge.

Reply:
I understand it's hardcoded temporary data, but we should still match the shape of the response because that's what will end up getting used in the components when we wire up the api.  Having this in sync with the api ensures the api response is a drop in replacement when we wire it up (and also helps catch any potential bugs in the meantime).